
# SurgicalTaskStateMsg.msg

uint8[2] robotMode